src/collections/Form/Form.js

```diff
@@ -167,96 +176,99 @@ function formSerializer(formNode) {
-      onSubmit={handleSubmit}
-      ref={(c) => (_form = _form || c)}
-    >
-      {children}
-    </ElementType>
-  )
-}
+  static propTypes = {
+    /** An element type to render as (string or function). */
+    as: customPropTypes.as,
 
-Form.Field = FormField
-Form.Button = FormButton
-Form.Checkbox = FormCheckbox
-Form.Dropdown = FormDropdown
-Form.Group = FormGroup
-Form.Input = FormInput
-Form.Radio = FormRadio
-Form.Select = FormSelect
-Form.TextArea = FormTextArea
-
-Form._meta = {
-  name: 'Form',
-  type: META.TYPES.COLLECTION,
-  props: {
-    widths: ['equal'],
-    size: _.without(SUI.SIZES, 'medium'),
-  },
-}
+    /** Primary content */
+    children: PropTypes.node,
 
-Form.propTypes = {
-  /** An element type to render as (string or function). */
-  as: customPropTypes.as,
+    /** Additional classes */
+    className: PropTypes.string,
 
-  /** Primary content */
-  children: PropTypes.node,
+    /** Automatically show any error Message children */
+    error: PropTypes.bool,
 
-  /** Additional classes */
-  className: PropTypes.string,
+    /** Automatically show a loading indicator */
+    loading: PropTypes.bool,
 
-  /** Automatically show a loading indicator */
-  loading: PropTypes.bool,
+    /** Called with (event, jsonSerializedForm) on submit */
+    onSubmit: PropTypes.func,
 
-  /** Automatically show any success Message children */
-  success: PropTypes.bool,
+    /** Called onSubmit with the form node that returns the serialized form object */
+    serializer: PropTypes.func,
 
-  /** Automatically show any error Message children */
-  error: PropTypes.bool,
+    /** A form can vary in size */
+    size: PropTypes.oneOf(_meta.props.size),
 
-  /** Automatically show any warning Message children */
-  warning: PropTypes.bool,
+    /** Automatically show any success Message children */
+    success: PropTypes.bool,
 
-  /** A form can vary in size */
-  size: PropTypes.oneOf(Form._meta.props.size),
+    /** Automatically show any warning Message children */
+    warning: PropTypes.bool,
 
-  /** Forms can automatically divide fields to be equal width */
-  widths: PropTypes.oneOf(Form._meta.props.widths),
+    /** Forms can automatically divide fields to be equal width */
+    widths: PropTypes.oneOf(_meta.props.widths),
+  }
 
-  /** Called onSubmit with the form node that returns the serialized form object */
-  serializer: PropTypes.func,
+  static _meta = _meta
 
-  /** Called with (event, jsonSerializedForm) on submit */
-  onSubmit: PropTypes.func,
-}
+  static Field = FormField
+  static Button = FormButton
+  static Checkbox = FormCheckbox
+  static Dropdown = FormDropdown
+  static Group = FormGroup
+  static Input = FormInput
+  static Radio = FormRadio
+  static Select = FormSelect
+  static TextArea = FormTextArea
 
-Form.defaultProps = {
-  as: 'form',
-  serializer: formSerializer,
-}
+  _form = null
 
-export default Form
+  handleRef = (c) => (this._form = this._form || c)
```

Comment:
This seems correct for now.  Eventually, the serializer will not be DOM dependent and this ref will go away.  I would suggest just setting it and avoiding the developer indirection however.

```jsx
ref={c => (this._form = this._form || c)}
```